app/src/main/res/drawable/ic_close.xml

```diff
@@ -8,6 +8,6 @@
     android:viewportWidth="24"
     android:viewportHeight="24">
     <path
-        android:fillColor="?primaryText"
```

Comment:
We can simply change this icon to `?neutral` since it's used in multiple places beyond the share sheet. We want to specify a constant fill color attribute for all our icons and override them manually by code for this reasoning.